pkgs/applications/window-managers/sway/contrib.nix

```diff
@@ -58,21 +66,20 @@ grimshot = stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "A helper for screenshots within sway";
-    homepage = "https://github.com/swaywm/sway/tree/master/contrib";
+    homepage = "https://github.com/OctopusET/sway-contrib";
     license = licenses.mit;
     platforms = platforms.all;
-    maintainers = sway-unwrapped.meta.maintainers ++ (with maintainers; [ evils ]);
+    maintainers = with maintainers; [ evils ];
```

Comment:
This does change the maintainers, as you noted in the meantime but I'm fine/happy with it.
@Synthetica9 is this fine with you as well?